---
title: "Adria Mitjavila - CV"
author: Adrià Mitjavila Ventura
output:
  pagedown::html_resume:
    css: ['css/styles.css', 'resume']
    # set to true to be able to see it in the navigator; false to visualize in the RStudio viewer
    self_contained: true
# this will produce the pdf output directly
knit: pagedown::chrome_print
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(
  results = "asis", 
  echo = FALSE,
  warning = FALSE,
  message = FALSE
)

library(tidyverse)
library(fontawesome)
```


# Aside

<center>
```{r logo, warning = FALSE, message = FALSE, error = FALSE, out.width="75%", eval=T, fig.align="center"}
knitr::include_graphics(path = "fotojo.png")
```
</center>

## Contact {#contact}

-   `r fontawesome::fa(name = "home")` Girona (Spain)
-   `r fontawesome::fa(name = "at")` [adria.mitjavila94\@gmail.com](mailto:adria.mitjavila94@gmail.com){.email}
-   `r fontawesome::fa(name = "globe")` [amitjavilaventura.github.io](https://amitjavilaventura.github.io){target="_blank"}
-   `r fontawesome::fa(name = "github")` [/amitjavilaventura](https://github.com/amitjavilaventura){target="_blank"}
-   `r fontawesome::fa(name = "orcid")` [0000-0002-0337-1929](https://orcid.org/0000-0002-0337-1929){target="_blank"}
-   `r fontawesome::fa(name = "blog")` [biobit.netlify.app](https://biobit.netlify.app){target="_blank"}


## Skills {#skills}

**Bioinformatics:**

-   Microarray, (small) RNA- / ChIP- / ATAC- / MNase-seq <!--, ENSEMBL, UCSC Genome Browser, IGV, NCBI Blast -->

**Informatics:**

-   R/Bioconductor, R Markdown, Shiny, Bash, Snakemake, Python, Git, HPC, Conda, Docker, Singularity <!--, Linux (Ubuntu), Mac OS, Windows, MSOffice, LibreOffice -->

**Experimental:**

-   (RT-)(q)PCR, primer design, electrophoresis, Western Blot, MNase-digestion, bacterial culture

**Languages:**

-   Catalan (Native), Spanish (Native), English (C1 - IELTS 7.5), Italian (Basic)

## Code

**[`chromHMMviewR`](https://amitjavilaventura.github.io/chromHMMviewR/)**<br>
**[`plotmics`](https://amitjavilaventura.github.io/plotmics/)**<br>
**[MYomics](https://github.com/amitjavilaventura/DPomics/tree/MYomics)** (*ongoing*)<br>
**[ATAC-seq snakemake](https://github.com/amitjavilaventura/ATACseq-snakemake)** (*ongoing*)<br>
More on [my GitHub profile](https://github.com/amitjavilaventura)


## Disclaimer {#disclaimer}

References available upon request.<br>
Build with [**pagedown**](https://github.com/rstudio/pagedown), code [here](https://github.com/amitjavilaventura/CVs/blob/main/cv-pagedown/amitjavila_resume.Rmd).<br>
Last updated on `r Sys.Date()`.


# Main

## Adrià Mitjavila Ventura {#title}

### Bioinformatician at *Institut Josep Carreras* | MSc

<p style="font-size:13px; text-align:justify;">Biotechnologist living around computers, interested in data analysis to support experimental research as well as in pure (bio)computational experiments and in software/pipeline development. See more about me on [my website](https://amitjavilaventura.github.io).</p>

## Experience {data-icon="laptop"}

### [Bioinformatician](https://amitjavilaventura.github.io/research/005-ijc/){target="_blank"}

*Institut Josep Carreras*, **Vavouri's lab**

Badalona, Spain

08/2021 - Present

<span style="font-size:9px">
Study of PIWI-interacting RNAs (piRNAs), their biogenesis mechanisms and their conservation across species.
</span>

### [Bioinformatician](https://amitjavilaventura.github.io/research/003-ieo/){target="_blank"}

*Istituto Europeo di Oncologia*, **Pasini's lab**

Milan, Italy

07/2019 - 07/2021

<span style="font-size:9px">
Study of Wnt/$\beta$catenin signaling pathway -and others- in the context of colorectal cancer and stem cell development.   |   Analysis and integration of RNA-seq, ChIP-seq and ATAC-seq data.
</span>

### [Master's thesis student & Research assistant](https://amitjavilaventura.github.io/research/002-ibis/){target="_blank"}

*Instituto de Biomedicina de Sevilla*, **Gene expression**

Seville, Spain

09/2017 - 10/2018

<span style="font-size:9px">
Study of Xrn1 and Ccr4 roles in yeast nucleosome architecture and gene transcription.   |   Analysis of MNase-seq data.   |   Performance and analysis of MNase-qPCR experiments.
</span>

### [Internship & Bachelor's thesis student](https://amitjavilaventura.github.io/research/001-microbial/){target="_blank"}

*Microbial. Sistemes i aplicacions analítiques, S.L.*

Girona, Spain

09/2016 - 06/2017

<span style="font-size:9px">
Microbiological and molecular analyses for pathogen detection and food forensics.   |   Study of butyrate effects on the ileal microbiota of colectomized colorectal cancer patients.
</span>

## Education {data-icon="graduation-cap" data-concise="true"}

### [MSc in Bioinformatics and Biostatistics](https://estudios.uoc.edu/es/masters-universitarios/bioinformatica-bioestadistica/presentacion){target="_blank"}

Universitat Oberta de Catalunya

Online

2018 - Present

<!--
<span style="font-size:9px">
Master's thesis: *xxx*.
</span>
-->

### [MSc in Molecular Genetics and Biotechnology](https://masteroficial.us.es/gmb/){target="_blank"}

Universidad de Sevilla - *Grade: 9.33/10*

Seville, Spain

2017 - 2018

<span style="font-size:9px">
Master's thesis: *Influence of Xrn1 in nucleosome positioning across <ins>Saccharomyces cerevisiae</ins> genome and its effects in transcription*. Grade: 9.2/10
</span>

### [BSc in Biotechnology](https://www.udg.edu/en/estudia/Oferta-formativa/Graus/Fitxes?IDE=993&ID=3103G0215){target="_blank"}

Universitat de Girona - *Grade: 8.12/10*

Girona, Spain

2012 - 2017

<span style="font-size:9px">
Bachelor's thesis: *Butyrate-induced changes in the diversity of intestinal mucosa-associated microbiota in colorectal cancer patients submitted to a lateral ileostomy*. Grade: 9.4/10
</span>

## Publications [`r fontawesome::fa(name = "orcid")`](https://orcid.org/0000-0002-0337-1929) {data-icon="book"}

### <span style="font-size:12px"> Xrn1 influence on gene transcription results from the combination of general effects on elongating RNA pol II and gene-specific chromatin configuration</span>

Begley V *et al*. RNA Biology. DOI: [10.1080/15476286.2020.1845504](https://doi.org/10.1080/15476286.2020.1845504){target="_blank"}.

N/A

2020